### PR TITLE
Added map of Honduras

### PR DESCRIPTION
**Pull request by avalladaresm**

### CODE REVIEW

File: packages/honduras/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# 🇭🇳 Honduras
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the states of Honduras:
```
Comment (VictorCazanave) [inline packages/honduras/README.md:5]: It seems that they are called "departments" (source: https://en.wikipedia.org/wiki/Departments_of_Honduras)

File: packages/honduras/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### yarn
+
+`yarn add @svg-maps/honduras`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/honduras) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
```
Comment (VictorCazanave) [inline packages/honduras/README.md:39]: Could you list the modifications done on the original map please?

File: packages/honduras/honduras.svg [hunk 3]
```diff
@@ -0,0 +1,96 @@
+<svg
+	xmlns="http://www.w3.org/2000/svg"
+	viewBox="0 0 711 522"
+	aria-label="Map of Honduras"
+>
+  <path
```
Comment (VictorCazanave) [inline packages/honduras/honduras.svg:6]: Could you use tab indentation in the whole file please?

**Comment by avalladaresm** [comment]
Hey @VictorCazanave, thanks for pointing out those issues. I have corrected them now, please take a look! :smiley:

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @avalladaresm for code

**Pull request merged**